### ISSUE TITLE
SCP Receiving Transfer Syntax

### ISSUE THREAD

**Issue by hewittwill** [PantelisGeorgiadis/dcmjs-dimse#7]
Does the SCP support receiving transfer syntaxes other than ImplicitLittleEndian? It appears where the Dataset is constructed it is not passed a Transfer Syntax, so defaults to ImplicitLittleEndian.

**Comment by PantelisGeorgiadis** [comment]
Hello @hewittwill! Thank you for your question. The SCP supports receiving all transfer syntaxes, since it doesn't do any transcoding (bit-streaming). Have in mind that you have to accept the transfer syntax you need, during the association request. Once you accept that syntax, the received dataset is constructed with it (check [Network.js, line 533](https://github.com/PantelisGeorgiadis/dcmjs-dimse/blob/240f371b8e3e1046729779dc289db5012d50f313/src/Network.js#L533)). I have just tested sending a JPEG2000 dataset from a third-party client, receiving it and verifying it.

**Comment by hewittwill** [comment]
Hey @PantelisGeorgiadis 

Sorry to clarify, what we're finding is whenever we save it to the filesystem, the file is substantially larger than when we sent it and (I think) is saving it as Explicit Little Endian. Keen to hear your thoughts!

Great library you've built though, we'd love to help

Thanks

**Comment by PantelisGeorgiadis** [comment]
Hello @hewittwill! We are on the same page! For some reason, the sending application is using the "Explicit Little Endian" presentation context and that why you get files with that syntax. The library doesn't do any kind of compression/decompression. However, I found a potential bug that might prevented receiving encapsulated syntaxes and I fixed it in v0.0.11.

**Comment by hewittwill** [comment]
Hey Pantelis, thanks for that. 

We're getting the following error:

```
Invalid vr type  - using UN
Invalid vr type  - using UN
Invalid vr type  - using UN
Invalid vr type  - using UN
Invalid vr type  - using UN
Invalid vr type ´ - using UN
WARN: Error: Request more than currently allocated buffer
````

Whenever we try to send using a Transfer Syntax other than Implicit or Explicit Little Endian (the two defined in your constants file. This error was caused when we manually added the TransferSyntax for JPEG Lossless (`1.2.840.10008.1.2.4.70`).

**Comment by PantelisGeorgiadis** [comment]
Hi @hewittwill! Regarding the `Error: Request more than currently allocated buffer`, that was fixed in v0.0.11. Please try that version. Regarding the `Invalid vr type  - using UN`, that's coming from dcmjs. There's a good chance the datasets you are pushing to contain tags with invalid VRs or private tags that dcmjs can't handle. What I observer is that the datasets are still arriving (and can be persisted to disk) but without these tags.

**Comment by hewittwill** [comment]
Ah thanks @PantelisGeorgiadis that fixed it! Sorry two different testing environments, updated in one place but not the other.
